pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -69,8 +78,21 @@ type CronWorkflowStatus struct {
 	LastScheduledTime *metav1.Time `json:"lastScheduledTime" protobuf:"bytes,2,opt,name=lastScheduledTime"`
 	// Conditions is a list of conditions the CronWorkflow may have
 	Conditions Conditions `json:"conditions" protobuf:"bytes,3,rep,name=conditions"`
+	// Succeeded is a counter of how many times the child workflows had success
+	Succeeded int64 `json:"succeeded" protobuf:"varint,4,rep,name=succeeded"`
+	// Failed is a counter of how many times a child workflow terminated in failed or errored state
+	Failed int64 `json:"failed" protobuf:"varint,5,rep,name=failed"`
+	// Phase defines the cron workflow phase. It is changed to Stopped when the stopping condition is achieved which stops new CronWorkflows from running
```

Comment:
This one was a bit incorrect, as it stops new Workflows from being scheduled, not CronWorkflows

```suggestion
	// v3.6 and after: Phase is an enum of Active or Stopped. It changes to Stopped when stopStrategy.condition is true
```
- add version specifier
- simplify a bit, similar to `StopStrategy` above